config.md

```diff
@@ -25,8 +25,9 @@ NOTE that there is no guarantee for forward or backward compatibility for versio
 
 Each container has exactly one *root filesystem*, specified in the *root* object:
 
-* **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
-* **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+* **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  A directory MUST exist at the path declared by the field.
+* **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container, defaults to false.
```

Comment:
> I liked “Defaults to false” better as its own sentence

I think it makes sense to combine them in one sentence, and it looks simpler, I'll change if more people think we should keep its own sentence.

> And defaults are a general enough idea that it may make more sense to mark them up (e.g. “readonly (bool, default: false) If true, then …”). So settings would either be required or have a default value.

Since it's a general change for all such defaults usage in runtime-spec, I'll leave it as a separate discussion if needed.